packages/server/src/health/service.ts

```diff
@@ -81,11 +81,20 @@ export class HealthService extends Service {
   }
 
   // TODO: move this somewhere else
-  private async callWebhook(url: string, data: any) {
+  private async callWebhook(url: string, data: any, token?: string) {
     const password = process.env.INTERNAL_PASSWORD || this.configService.current.security?.password
+    const config: AxiosRequestConfig = { headers: {} }
+
+    if (password) {
+      config.headers.password = password
+    }
+
+    if (token) {
+      config.headers['X-Webhook-Token'] = token
```

Comment:
You are using uppercase `X-Webhook-Token` here and lowercase `x-webhook-token` in the main repo. Can you make this lowercase for consistency?